cypress/support/commands/tradersHub.js

```diff
@@ -62,10 +62,13 @@ Cypress.Commands.add('c_completeTradersHubTour', (options = {}) => {
 })
 
 Cypress.Commands.add('c_enterValidEmail', (signUpMail, options = {}) => {
-  const { language = 'english' } = options
+  const { language = 'english', size = 'desktop' } = options
```

Comment:
lets default to mobile for this as well
```suggestion
  const { language = 'english', size = 'mobile' } = options
```